src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -0,0 +1,217 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState, useRef } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import PointerBox, {
+  Position
+} from 'src/shared/components/pointer-box/PointerBox';
+import SpeciesStats from 'src/content/app/species/components/species-stats/SpeciesStats';
+import ExpandableSection from 'src/shared/components/expandable-section/ExpandableSection';
+
+import {
+  getActiveGenomeId,
+  getActiveGenomeStats
+} from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getGenomeExampleFocusObjects } from 'src/shared/state/genome/genomeSelectors';
+import { fetchStatsForActiveGenome } from 'src/content/app/species/state/general/speciesGeneralSlice';
+
+import { RootState } from 'src/store';
+import { GenomeStats } from '../../state/general/speciesGeneralSlice';
+import ViewInApp, { urlObj } from 'src/shared/components/view-in-app/ViewInApp';
+import {
+  StatsSection,
+  sectionGroupsMap
+} from '../../state/general/speciesGeneralHelper';
+import { ExampleFocusObject } from 'src/shared/state/genome/genomeTypes';
+
+import styles from './SpeciesMainView.scss';
+
+type Props = {
+  activeGenomeId: string | null;
+  genomeStats: GenomeStats | undefined;
+  exampleFocusObjects: ExampleFocusObject[];
+  fetchStatsForActiveGenome: () => void;
+};
+
+const ExampleLinks = (props: {
+  exampleLinks?: Partial<urlObj>;
+  exampleLinkText?: string;
+}) => {
```

Comment:
Several considerations:
- The component appears as a single link, although it might show the popup with buttons (compare our other ExampleLinks components, where we are showing multiple links). Perhaps better call it `ExampleFeatureLink` or `ViewExample`? Is it possible that this is a reusable component? It doesn't look particularly specific to `SpeciesMainViewStats`.
- The `example` part of the `exampleLinks` prop is redundant. `links` should be sufficient, as it is called in the `ViewInApp` component.
- The `exampleLinkText` can then simply become `children`
- When there is only one app in which the example can be viewed (e.g. example region can only be viewed in GenomeBrowser), I am sure there is no point in showing the popup. We should just redirect the user there.

Reply:
> Is it possible that this is a reusable component? It doesn't look particularly specific to SpeciesMainViewStats.

How about creating something like `ViewInAppPopup`?

Reply:
Sounds good.